### ISSUE TITLE
Snapshottest does not function correctly when --parallel is used

### ISSUE THREAD

**Issue by rekahsoft** [syrusakbary/snapshottest#56]
When tests are run in parallel, snapshottest writes the resulting snapshot to a file based on the name of the python source file. For example, `test_something.py` results in `snap_test_something.py`. This however seems to lead to race conditions when a file contains more then one snapshot test and tests are run in parallel using `./manage.py test --parallel 4` which has different resulting snapshots then when running `./manage.py test`.

**Comment by paulmelnikow** [comment]
As I mentioned in #107, I wonder if the fix might be to provide a way to pass this option through an environment variable, which subprocesses would then automatically inherit.

**Comment by paulmelnikow** [comment]
> his however seems to lead to race conditions when a file contains more then one snapshot test and tests are run in parallel using `./manage.py test --parallel 4` which has different resulting snapshots then when running `./manage.py test`.

Right, so this is the problem. It's not about passing the option through.

Hmm, I wonder if we could add an option where each snapshot is written to its own file. I think that's the only real way to solve this.

**Comment by zeroindex** [comment]
I'm curious if anything ever came of this. Is there a flag to get each snapshot into its own file?
